server/controllers/index.js

```diff
@@ -78,7 +79,13 @@ if (process.env.NODE_ENV === 'testing') {
 /* Current player plays a card and a word */
 router.post('/api/playCardWord', auth, (req, res) => {
     gameLogic.playCardAndWord(req.session.user, req.body.card, req.body.word);
-    res.sendStatus(200);
+    
+    if (!profanities.notProfanity(req.body.word)) {
+        res.sendStatus(200);
+        console.log("Profane");
```

Comment:
I might be wrong but wouldn't "profane" mean there _is_ profanity?

Reply:
[Wikipedia](https://en.wikipedia.org/wiki/Profanity) seems to agree.

Reply:
yep! have since changed this, it's because at first I thought word-list contained bad words but it infact contains good words

Reply:
Oh, I see : )